astroquery/alma/core.py

```diff
@@ -280,33 +287,50 @@ def stage_data(self, uids):
                     # this indicates a wrong server is being used;
                     # the "pre-feb2020" stager will be phased out
                     # when the new services are deployed
-                    return self.stage_data_prefeb2020(uids)
+                    raise ValueError("Failed query!  This shouldn't happen - please "
```

Comment:
Should this be ServerError, or one of the more specific errors we have,  instead plain ValueError? (Given the user is unlikely can do anything about it)

Reply:
yes